ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -843,6 +850,12 @@ private AdminResponse handleBlobStoreControlRequest(DataInputStream requestStrea
                   if (storageManager.shutdownBlobStore(partitionId)) {
                     error = ServerErrorCode.No_Error;
                     logger.info("Store is successfully shutdown for partition: {}", partitionId);
+                    List<PartitionId> failToUpdateList =
+                        storageManager.setBlobStoreStoppedState(Arrays.asList(partitionId), true);
+                    if (!failToUpdateList.isEmpty()) {
+                      logger.info("Fail to add BlobStore(s) {} to stopped list after stop operation completed",
```

Comment:
same here. WARN level if this is unexpected.

Reply:
fixed